### ISSUE TITLE
0.7.0@beta8 tries to symlink an already-existing directory

### ISSUE THREAD

**Issue by BrianHicks** [rtfeldman/elm-css#317]
I'm getting this in CI:

```
running `$WORKSPACE/node_modules/.bin/elm-css --output $WORKSPACE/app/assets/stylesheets/elm/fast --pathToMake $WORKSPACE/ui/node_modules/.bin/elm-make`
fs.js:1034
  return binding.symlink(preprocessSymlinkDestination(target, type, path),
                 ^

Error: EEXIST: file already exists, symlink '$WORKSPACE/ui/css/elm-stuff' -> '$WORKSPACE/ui/elm-stuff/generated-code/rtfeldman/elm-css/elm-stuff'
    at Object.fs.symlinkSync (fs.js:1034:18)
    at module.exports ($WORKSPACE/node_modules/elm-css/index.js:54:8)
    at Object.<anonymous> ($WORKSPACE/node_modules/elm-css/js/cli.js:27:1)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
    at Module.runMain (module.js:605:10)
    at run (bootstrap_node.js:427:7)
```

Looks like 0.7.0@beta8 is trying to symlink `elm-stuff`, but a symlink already exists.

**Comment by ahstro** [comment]
This seems to have been fixed in b5844f4658bcbd3d893be0d9ec7fa06f0c2df5e8, although it's not yet published to npm.

**Comment by rtfeldman** [comment]
Closing in favor of #327
